PDCMFinder/cancer-models
# 81 dev ops deploy website to devcancermodelsorg

## Issue
Implements CI/CD to dev.cancermodels.org

## Description
Changes to deployment to point to the k8s descriptors mapped to dev.cancermodels.org

## Testing instructions
Make a change and deploy on GitLab and check the change is in dev.cancermodels.org

## Screenshots (optional)
